azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/implementation/ResourceManager.java

```diff
@@ -149,7 +149,8 @@ public ResourceManager withSubscription(String subscriptionId) {
     }
 
     private ResourceManager(RestClient restClient, String subscriptionId) {
-        super(restClient, subscriptionId);
+        super(null, subscriptionId);
```

Comment:
ideally, I think it might make sense to just have all the other managers derive from ResourceManager as their base class, at least on the impl side, but that requires some more thought too - not for beta2..